src/docs/style/wiredtiger11.css

```diff
@@ -0,0 +1,187 @@
+/* WiredTiger-specific styling. */
+
+@media (prefers-color-scheme: dark) {
+  body {
+    --page-foreground-color: #CCC;
```

Comment:
Where are these variables used?

Are these Doxygen variables, or does CSS now understand these variable names? If so, if you have a good link explaining this, that would be much appreciated.

Reply:
This is moved to a new file `wiredtiger11.css` because older versions don't support dark mode and styling breaks when there's only a partial support.